pkg/epp/flowcontrol/types/request.go

```diff
@@ -82,29 +84,20 @@ type QueueItemHandle interface {
 //
 // The Flow Controller creates an object that implements this interface by wrapping an incoming `FlowControlRequest`.
 type QueueItemAccessor interface {
-	// EnqueueTime is the timestamp when the item was logically accepted by the Flow Controller for queuing (i.e., when
-	// `FlowController.EnqueueAndWait()` was called.
-	EnqueueTime() time.Time
-
-	// ByteSize returns the byte size of the original request, cached from `FlowControlRequest.ByteSize()`.
-	ByteSize() uint64
-
-	// FlowID returns the unique identifier of the flow this item belongs to, cached from `FlowControlRequest.FlowID()`.
-	FlowID() string
-
-	// EffectiveTTL is the actual Time-To-Live assigned to this item by the Flow Controller, taking into account the
-	// request's preference (`FlowControlRequest.InitialEffectiveTTL()`) and any Flow Controller or per-flow
-	// defaults/policies.
-	EffectiveTTL() time.Duration
-
-	// RequestID is the user-facing ID from the original request (`FlowControlRequest.ID()`).
-	RequestID() string
-
```

Comment:
@kfswain These can all be obtained from the `OriginalRequest()`. I'm removing an unnecessary layer of indirection for simplicity and to scope down the interface.